api_spec_oa3.yml

```
@@ -58,34 +97,43 @@ paths:
             validate the existence of asset files like logos and
             screenshots.
       responses:
-        200:
-          description: |-
-            Validation Ok, return latest valid publiccode version
+        '200':
+          description: 'Validation Ok, return latest valid publiccode version'
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/PublicCode'
+                $ref: '#/components/schemas/Validation'
             application/x-yaml:
               schema:
-                $ref: '#/components/schemas/PublicCode'
-        422:
+                $ref: '#/components/schemas/Validation'
+        '422':
           description: Validation failed
-          content: 
+          content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/ValidationError'
+                $ref: '#/components/schemas/Validation'
 components:
   schemas:
     ValidationError:
       properties:
-        Key:
+        key:
           type: string
-        Reason:
+        reason:
           type: string
       required:
-        - Key
-        - Reason
+        - key
+    Validation:
+      properties:
+        status:
+          type: integer
```

Is this the http status?

```suggestion
        status:
          type: integer
          format: int32
```